# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../epplets16/e16conf-path.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- epplets-0.7/api/epplet.c.orig	2006-01-08 21:34:51.000000000 +0100
+++ epplets-0.7/api/epplet.c	2006-01-08 21:36:40.000000000 +0100
@@ -5497,7 +5497,7 @@
    pid_t               pid;
 
    /* make sure basic dir exists */
-   Esnprintf(s, sizeof(s), "%s/.enlightenment/epplet_config", getenv("HOME"));
+   Esnprintf(s, sizeof(s), "%s/.e16/epplet_config", getenv("HOME"));
    if (stat(s, &st) < 0)
      {
 	if (mkdir(s, S_IRWXU) < 0)
@@ -5514,7 +5514,7 @@
      }
 
    /* make sure this epplets config dir exists */
-   Esnprintf(s, sizeof(s), "%s/.enlightenment/epplet_config/%s",
+   Esnprintf(s, sizeof(s), "%s/.e16/epplet_config/%s",
 	     getenv("HOME"), name);
    conf_dir = strdup(s);
    if (stat(s, &st) < 0)
--- epplets-0.7/epplets/E-Slides.c.orig	2006-01-08 21:35:37.000000000 +0100
+++ epplets-0.7/epplets/E-Slides.c	2006-01-08 21:36:58.000000000 +0100
@@ -569,7 +569,7 @@
 
   path = Epplet_query_config("image_dir");
   if (path == NULL) {
-    Esnprintf(buff, sizeof(buff), "%s/.enlightenment/backgrounds", getenv("HOME"));
+    Esnprintf(buff, sizeof(buff), "%s/.e16/backgrounds", getenv("HOME"));
     path = strdup(buff);
     Epplet_add_config("image_dir", buff);
   } else {
